integration-tests/utils/integration-utils.js

```diff
@@ -103,7 +103,7 @@ const clickTableCell = async (page, { tableName, columnHeader, textContains, isD
   return (await page.waitForXPath(xpath, options)).click()
 }
 
-const click = async (page, xpath, options) => {
+const click = async (page, xpath, options = { visible: true }) => {
```

Comment:
I had no idea that the default value of visible is `false` in `waitForXPath`! That seems like a poor choice on their part.

https://puppeteer.github.io/puppeteer/docs/9.1.1/puppeteer.page.waitforxpath/

That being said, it's probably worth auditing the rest of the calls to `waitForXPath`

Reply:
I'm very surprised we hadn't already been setting `visible` to `true`, which makes me want to check with @petesantos and/or @breilly2 in case there was a particular reason for it.